[Request][8.15 & Serverless] Alert suppression for ES|QL and ML rules (#5568)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

NOTE: If a required job isn't currently running, it will automatically start when you finish configuring and enable the rule.
.. The anomaly score threshold above which alerts are created.
+
. preview:[] (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Use *Suppress alerts by* to reduce the number of repeated or duplicate alerts created by the rule. Refer to <<alert-suppression>> for more information.
+

> NOTE: Because {ml} rules generate alerts from anomalies, alert documents don't contain source event fields and you can only use anomaly fields when configuring alert suppression.

Reviewer: I tried simplifying the phrasing a bit for clarity:

```suggestion
NOTE: Because {ml} rules generate alerts from anomalies, which don't contain source event fields, you can only use anomaly fields when configuring alert suppression.
```

- When writing an aggregating query, use the {ref}/esql-commands.html#esql-stats-by[`STATS...BY`] command with fields that you want to search and filter for after alerts are created. For example, using the `host.name`, `user.name`, `process.name` fields with the `BY` operator of the `STATS...BY` command returns these fields in alert documents, and allows you to search and filter for them from the Alerts table. 

> - When configuring alert suppression on a non-aggregating query, avoid sorting alerts by descending `@timestamp` order. Doing so can impact the number of alerts that are suppressed, especially if the number of alerts generated is higher than the maximum number of alerts the rule can create when it runs.

Reviewer 2: I would rephrase:
 ```suggestion
- When configuring alert suppression on a non-aggregating query, it is recommended to sort results by ascending `@timestamp` order. It will ensure the number of alerts will be suppressed, especially if the number of alerts generated is higher than the maximum number of alerts the rule can create when it runs.
```

Author: Updated in [893ab42](https://github.com/elastic/security-docs/pull/5568/commits/893ab42f59ed03a3488dc16698d940a137cdfda0)
